src/dmd/traits.d

```diff
@@ -86,7 +86,7 @@ private Dsymbol getDsymbolWithoutExpCtx(RootObject oarg)
     return getDsymbol(oarg);
 }
 
-private const StringTable traitsStringTable;
+private const StringTable!string traitsStringTable;
```

Comment:
No need to duplicate the key by the value, you can just use StringTable!bool and print `sub.lstring()` below.